common/src/main/java/io/netty/util/concurrent/AbstractFuture.java

```diff
@@ -15,44 +15,14 @@
  */
 package io.netty.util.concurrent;
 
-import java.util.concurrent.CancellationException;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
-
 /**
  * Abstract {@link Future} implementation which does not allow for cancellation.
  *
  * @param <V>
+ *
+ * @deprecated use {@link Future} directly.
  */
+@Deprecated
```

Comment:
Assuming this is for Netty5, given the JDK8 usage?  Any reason to mark this deprecated rather than removing it?

Reply:
nope let me remove it